web_responsive/static/src/js/web_responsive.js

```diff
@@ -4,7 +4,6 @@
 odoo.define('web_responsive', function(require) {
     'use strict';
 
-    var $ = require('$');
```

Comment:
[So it seems...](https://github.com/odoo/odoo/commit/d56b35e26e4b578c5f6f434d336edd006d321860#diff-dfbc96e4758b063efc18cb19c9719fc9L41)